ansible/roles/webworker/tasks/main.yml

```diff
@@ -1,23 +1,10 @@
 # Tasks to be run on machines running django
 
-- name: create restore payloads directory
```

Comment:
I moved these to the 'host' machine since it didn't seem to make sense to run them on all the webworkers when they are all looking at the same shared folders.